pluginconfig/pluginconfig-311.yaml

```diff
@@ -171,13 +171,13 @@ versions:
       genevaLogging: osarpint.azurecr.io/acs/mdsd:master.20190228.1
       genevaStatsd: osarpint.azurecr.io/acs/mdm:git-a909a2e76
       genevaTDAgent: osarpint.azurecr.io/acs/td-agent:master.20190228.1
-      azureControllers: quay.io/openshift-on-azure/azure:latest
-      canary: quay.io/openshift-on-azure/azure:latest
-      etcdBackup: quay.io/openshift-on-azure/azure:latest
-      metricsBridge: quay.io/openshift-on-azure/azure:latest
-      startup: quay.io/openshift-on-azure/azure:latest
-      sync: quay.io/openshift-on-azure/azure:latest
-      tlsProxy: quay.io/openshift-on-azure/azure:latest
+      azureControllers: quay.io/openshift-on-azure/azure:v10.0
+      canary: quay.io/openshift-on-azure/azure:v10.0
+      etcdBackup: quay.io/openshift-on-azure/azure:v10.0
+      metricsBridge: quay.io/openshift-on-azure/azure:v10.0
+      startup: quay.io/openshift-on-azure/azure:v10.0
+      sync: quay.io/openshift-on-azure/azure:v10.0
+      tlsProxy: quay.io/openshift-on-azure/azure:v10.0
```

Comment:
Note: MS has asked us to not switch to ACR this sprint as our current credentials to push an image don't work, and Julien is OOO for the holiday so it would delay the release a day.